metrics/prometheus-agent/values.yaml

```diff
@@ -0,0 +1,50 @@
+fullnameOverride: prometheus-agent
+
+global:
+  rbac:
+    create: true
+
+prometheus:
+  prometheusSpec:
+    enableAdminAPI: true
+    logFormat: json
+    logLevel: warn
+    scrapeInterval: 30s
+    serviceAccountName: prometheus-agent
+    remoteWrite:
+    - authorization:
+        credentials:
+          name: coralogix-keys
+          key: PRIVATE_KEY
+      name: prometheus-agent-coralogix
+      queueConfig:
+        capacity: 2500
+        maxSamplesPerSend: 1000
+        maxShards: 200
+      remoteTimeout: 120s
+      url: https://prometheus-gateway.eu2.coralogix.com:9090/prometheus/api/v1/write?external_labels=CX_LEVEL    
```

Comment:
Is it allowed through 443?

Reply:
The metrics are arriving their destination so yes